cost/azure/rightsize_managed_disks/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v1.2
+
+- Updated the descriptions and labels of the IOPS and throughput parameters in the README and policy template files.
+- Updated the short description of the policy.
+- Changed the functionality of `param_min_savings`: Before this version, the `param_min_savings` parameter was used to consider the total savings (the sum of all the savings per resource) and not the savings per resource to decide whether to recommend or not. In this new version, this parameter is used to recommend or not based on the savings of each resource, just as other policies do.
```

Comment:
Perhaps: "Fixed  the functionality of Minimum Savings Threshold parameter. This parameter is used to suppress recommendations with potential savings below specified threshold.

Reply:
thank you! done!